webapp/platform/client/src/client4.ts

```diff
@@ -7,7 +7,7 @@ import FormData from 'form-data';
 
 import {PreferenceType} from '@mattermost/types/preferences';
 import {SystemSetting} from '@mattermost/types/general';
-import {ClusterInfo, AnalyticsRow, SchemaMigration, LogFilter} from '@mattermost/types/admin';
+import {ClusterInfo, AnalyticsRow, SchemaMigration, LogFilter, LogFilterQuery} from '@mattermost/types/admin';
```

Comment:
nit: `LogFilter` is unused now (0/5, but main repo lints these)

Reply:
Good catch. I forgot we never added linting to the other packages. I should fix that one of these days...